doc/api/cli.md

```diff
@@ -574,6 +574,17 @@ Disable the `Object.prototype.__proto__` property. If `mode` is `delete`, the
 property is removed entirely. If `mode` is `throw`, accesses to the
 property throw an exception with the code `ERR_PROTO_ACCESS`.
 
+### `--disable-sigusr1`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.1 Active Development
```

Comment:
This stability index means that the feature is not finished. What is missing?

Reply:
My idea was to make it experimental just in case the flag name isn't clear, or the feature does contain some gaps not identified yet, so we could fix it in a semver-minor instead of a semver-major in the case of breaking changes.

Reply:
In that case, I think it should be 1.2:

```suggestion
> Stability: 1.2 - Release candidate
```